xml/FrameworksIndex/dotnet-plat-ext-2.1.xml

```diff
@@ -42,11 +42,11 @@
     <Assembly Name="System.Security.Permissions" Version="4.0.1.0" />
     <Assembly Name="System.Security.Principal.Windows" Version="4.1.1.0" />
     <Assembly Name="System.ServiceModel" Version="4.0.0.0" />
-    <Assembly Name="System.ServiceModel.Duplex" Version="4.1.0.0" />
-    <Assembly Name="System.ServiceModel.Http" Version="4.1.2.0" />
-    <Assembly Name="System.ServiceModel.NetTcp" Version="4.1.2.0" />
-    <Assembly Name="System.ServiceModel.Primitives" Version="4.2.0.0" />
-    <Assembly Name="System.ServiceModel.Security" Version="4.0.3.0" />
+    <Assembly Name="System.ServiceModel.Duplex" Version="4.6.0.0" />
```

Comment:
Can these additions to the 2.1 and 2.2 platform extensions be correct? They're all (or almost all) WCF changes.